README.md

```diff
@@ -47,7 +54,13 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+town[:residents].delete("Belle")
+town[:castle][:guests]<<["Belle"]
+
```

Comment:
👍  what about this: `town[:castle][:guests] << town[:residents].delete("Belle")`